### DISCUSSION TITLE
El botón de adjuntar o gps puede resaltarse más?

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#198] [2. Questions and Answers] [answered]
Hola

He visto los cambios en los formularios y son muy buenos, sobre todo la fecha y hora + el gps. Me he fijado que no resalta mucho el botón adjuntar, ya que el borde es muy pequeño y el fondo es el mismo que el formulario, hasta se puede perder de vista.

Se le podría agregar algún fondo o engrosar las letras y bordes?

Adjunto imagen
![image](https://user-images.githubusercontent.com/61419471/118831294-a7cc0f80-b88d-11eb-988f-5ab9e8b6b16c.png)

Gracias :>

**Comment by gmoplaza** [comment, marked as answer]
Hola @NicolasPLG todos los diseños nuevos fueron analizados antes de la implementación, obviamente estamos abiertos a recibir todo el feedback que nos den :), pero por ahora nos vamos a dedicar al desarrollo de otras funcionalidades.
De todas formas si con el tiempo esto genera problemas nos cuentas de nuevo y volvemos a analizar, pero demos un tiempo para probar los nuevos formularios.
Gracias por tus comentarios.

**Reply by NicolasPLG** [reply]
Hola @gmoplaza muchas gracias por la respuesta
